highway/client/cosmos.go

```diff
@@ -80,9 +108,9 @@ func (cc *Cosmos) QueryAllNames() ([]types.WhoIs, error) {
 }
 
 // QueryAllNames returns all names registered on the blockchain
-func (cc *Cosmos) QueryName(name string) (*types.WhoIs, error) {
+func (cc *Cosmos) QueryName(name string) (*rt.WhoIs, error) {
 	// query the blockchain using the client's `WhoIsAll` method to get all names
-	queryResp, err := cc.query.WhoIs(context.Background(), &types.QueryGetWhoIsRequest{
+	queryResp, err := cc.registryQuery.WhoIs(context.Background(), &rt.QueryGetWhoIsRequest{
```

Comment:
should we be nill checking the background on context? or is garanteed

Reply:
it returns. anon-nil empty context, this is gaurenteed